### DISCUSSION TITLE
How to get access_token from shopify store

### DISCUSSION THREAD

**Discussion by olegpokhylo** [pwwang/liquidpy#55] [Q&A] [answered]
I am developing the shopify bot with Python. I got the access_token from Shopify store but api request is failed every time.
I copied the access_token correctly but don't know why it is happened.

**Comment by lozovskijaleksej** [comment, marked as answer]
You should define the access scope of your api_key.
![image](https://github.com/pwwang/liquidpy/assets/153434083/a35d2aaa-9eaa-4bf3-a2bf-6acc4e7efe25)
Refer this image and try to do it.

**Reply by olegpokhylo** [reply]
It works well. Thanks for your help :)
